### ISSUE TITLE
GraalJS fails to install in CI

### ISSUE THREAD

**Issue by Ms2ger** [tc39/test262#3692]
```
#!/bin/bash -eo pipefail
./node_modules/.bin/esvu install $hostName

esvu ❯ version 1.2.9
GraalJS ❯ Checking version...
GraalJS ❯ Installing version 21.3.3.1
GraalJS ❯ Downloading https://github.com/oracle/graaljs/releases/download/vm-21.3.3.1/graaljs-21.3.3.1-linux-amd64.tar.gz
progress [----------------------------------------] 0% | ETA: 0s | 0/47315757progress [===========-----------------------------] 27% | ETA: 1s | 13080704/47315757progress [=========================---------------] 62% | ETA: 0s | 29720576/47315757progress [====================================----] 90% | ETA: 1s | 42713088/47315757progress [========================================] 100% | ETA: 0s | 47315757/47315757GraalJS ❯ Extracting /tmp/esvu-ddf2ac3d55c5d5465e2a64c5ee2ee159
GraalJS ❯ Installing /tmp/esvu-ddf2ac3d55c5d5465e2a64c5ee2ee159-extracted
esvu ✖ [Error: ENOENT: no such file or directory, copyfile '/tmp/esvu-ddf2ac3d55c5d5465e2a64c5ee2ee159-extracted/graaljs-21.3.3.1-linux-amd64/lib/libjsvm.so' -> '/home/circleci/.esvu/engines/graaljs/graaljs-21.3.3.1-linux-amd64/lib/libjsvm.so'] {
  errno: -2,
  code: 'ENOENT',
  syscall: 'copyfile',
  path: '/tmp/esvu-ddf2ac3d55c5d5465e2a64c5ee2ee159-extracted/graaljs-21.3.3.1-linux-amd64/lib/libjsvm.so',
  dest: '/home/circleci/.esvu/engines/graaljs/graaljs-21.3.3.1-linux-amd64/lib/libjsvm.so'
}
esvu ✖ Some engines were not installed.

Exited with code exit status 1

CircleCI received exit code 1
```

cc @devsnek

**Comment by devsnek** [comment]
@Ms2ger i've published esvu 1.2.10 to fix this

**Comment by ptomato** [comment]
Thanks, seems to work: https://github.com/tc39/test262/pull/3693
